assets/blocks/course-list-block/course-list.scss

```diff
@@ -112,8 +112,11 @@ $block: '.wp-block-sensei-lms-course-list';
 				}
 			}
 
-			.sensei-block-wrapper {
+			.sensei-block-wrapper, form[action^="?add-to-cart"] {
```

Comment:
I've noticed that the form action changes when you add the product to the cart. Is there a better way to target the purchase form element? Is it possible to wrap it in a `<div class="sensei-block-wrapper">`?

![image](https://user-images.githubusercontent.com/1612178/203562993-73335317-8c0a-4475-924b-3cc46e932dd1.png)

Reply:
Thanks, updated it here https://github.com/Automattic/sensei-pro/pull/2078